examples/quickstart.py

```diff
@@ -12,8 +12,8 @@
 from imitation.util import logger, util
 
 # Load pickled test demonstrations.
-with open("tests/data/expert_models/cartpole_0/rollouts/final.pkl", "rb") as f:
-    # This is a list of `imitation.data.types.Trajectory`, where
+with open("tests/testdata/expert_models/cartpole_0/rollouts/final.pkl", "rb") as f:
+    # This is a list of `imitation.testdata.types.Trajectory`, where
```

Comment:
This is one replace to many, right? (should still be `imitation.data.types.Trajectory`)

Reply:
Yep, good catch.